crates/starknet-rs-py/src/starknet_state.rs

```diff
@@ -30,10 +30,12 @@ impl PyStarknetState {
     pub fn declare(
         &mut self,
         contract_class: &PyContractClass,
+        hash_value: Option<BigUint>,
```

Comment:
I strongly recommend here something like:
```suggestion
        hash_value: Option<[u8; 32]>,
```
And to standardize that to a little endian representation.

Reply:
yes it is a better approach